keyboards/handwired/yamsek/info.json

```
@@ -0,0 +1,73 @@
+{
+    "manufacturer": "Johannes Schneider",
+    "keyboard_name": "YaMseK",
+    "maintainer": "Johannes Schneider",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "url": "https://hackaday.io/project/21164-yamsek",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x6060",
+        "vid": "0xFEED"
+    },
+    "layouts": {
```

Because you have a custom matrix, you need to specify the matrix size somewhere (for the default matrix these values are inferred from `matrix_pins.cols` and `matrix_pins.rows`, but for a custom matrix there is no way to autodetect them).
```suggestion
    "matrix_size": {
        "cols": 6,
        "rows": 8
    },
    "layouts": {
```